util/src/main/java/io/kubernetes/client/Exec.java

```diff
@@ -349,7 +349,16 @@ static int parseExitCode(ApiClient client, InputStream inputStream) {
     return -1;
   }
 
-  protected static class ExecProcess extends Process {
+  public interface ResizableStream {
```

Comment:
What is the purpose of adding these interfaces? Why not just add the methods to `ExecProcess`?

Reply:
> What is the purpose of adding these interfaces?

the standard interface `java.lang.Process` only exposes access to three streams: stdin/stdout/stderr. while for a complete implementation of "web-terminal", we will have to leave the developers public access to another stream named "resize" which tells the kubernetes exec stream to probably adjust the terminals width&height.. 

> Why not just add the methods to ExecProcess?

it  is protected. otherwise how do you think about promote it to a public inner class?

Reply:
I guess my feeling is you're going to have to cast it one way or the other:

```java
Process p = Exec.exec(...);
ResizableExecStream res = (ResizableExecStream)p;
res.getResizeStream();
...
```

vs

```java
ExecProcess p = (ExecProcess) Exec.exec(...);
p.getResizeStream();
```

I don't see the win from the interfaces, but I could be wrong.

Reply:
i reverted the interfaces and made the class `ExecProcess` public w/ the new getters, wdyt?